neuromation/cli/job.py

```diff
@@ -307,7 +307,22 @@ def job() -> None:
 ) -> None:
     """
     Execute command in a running job.
+
+    Examples:
+
+    # Provides a shell to the container via `/bin/bash`:
+    neuro exec my-job
+    neuro exec --tty my-job /bin/bash
+
+    # Executes a single command in the container and returns the control:
+    neuro exec my-job ls > ls_output.txt && echo "OK"
+    neuro exec my-job /bin/not-an-executable || echo "failed"
     """
+    # TODO (artem) Provide a backup option if the container lacks `/bin/bash`
```

Comment:
something like:
try: run `/bin/bash`, except executable-not-found-error: run `/bin/sh` ...

But definitely later, not in this PR. Probably this concern is pointless since (1) most of data scientists' containers are based on ubuntu or are reach and have `/bin/bash` anyhow, and (2) we explicitly stated that by default we run `/bin/bash` in the help message, so the user is aware.